### ISSUE TITLE
Dynamic prompts not working

### ISSUE THREAD

**Issue by WilliamChelman** [terkelg/prompts#53]
Just trying to reproduce the [example](https://www.npmjs.com/package/prompts#dynamic-prompts) and I have the error
```
Error: prompt type (prev => (prev == 'pizza' ? 'text' : null)) is not defined
    at prompt (/container-init/node_modules/prompts/lib/index.js:43:13)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
```

Great work on this library none the less 😺 

EDIT: 
node version: 8.9.4

**Comment by terkelg** [comment]
Can you please share your code? It looks like it's trying to find a prompt of type 
```js
(prev => (prev == 'pizza' ? 'text' : null))
```

**Comment by terkelg** [comment]
I found the bug. Thanks for reporting this! I'll fix this asap.

**Comment by terkelg** [comment]
Fixed in 225818b23e4e862fe3bd93580f4729a3cefdf3d1 and new version released. 🎉
